File: wear/src/main/java/com/thewizrd/simplewear/LaunchActivity.java
package com.thewizrd.simplewear;

import android.content.Intent;
import android.os.Bundle;
import android.support.wearable.activity.WearableActivity;

public class LaunchActivity extends WearableActivity {

    private static final String TAG = "LaunchActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        setTheme(R.style.WearAppTheme);
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, PhoneSyncActivity.class)
                .setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);

        // Navigate
        startActivity(intent);
        finishAffinity();
    }
}

File: wear/src/main/java/com/thewizrd/simplewear/LaunchActivity.kt
package com.thewizrd.simplewear

import android.app.Activity
import android.content.Intent
import android.os.Bundle

class LaunchActivity : Activity() {
    companion object {
        private const val TAG = "LaunchActivity"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this, PhoneSyncActivity::class.java)
                .setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK)

        // Navigate
        startActivity(intent)
        finishAffinity()
    }
}